packages/sanity/src/router/IntentLink.tsx

```diff
@@ -3,16 +3,42 @@ import {IntentParameters} from './types'
 import {useIntentLink} from './useIntentLink'
 
 /**
+ * The props for the `IntentLink` component that creates a link to an intent.
+ *
  * @public
  */
 export interface IntentLinkProps {
+  /**
+   * The name of the intent to link to.
+   */
   intent: string
+
+  /**
+   * The parameters to include in the intent.
+   */
   params?: IntentParameters
+
+  /**
+   * Whether to replace the current URL in the browser history instead of adding a new entry.
+   */
   replace?: boolean
 }
 
 /**
+ * A component that creates a link to an intent.
+ *
  * @public
+ *
+ * @param props - Props to pass to the anchor element.
```

Comment:
here it would make sense to link to `IntentLinkProps` but it also doesn't make sense to hardcode a url with a tag. I am going to see if we can use relative links instead such as 

```ts
/**
* {@link /router/IntentLinkProps | IntentLinkProps}
*/
```

Reply:
I just saw this now @binoy14 , apart from the complications with links overall, did you look into the relative ones? 👀

Reply:
No, since I am not sure how the links will come thru I didn't get to the relative links yet. If it makes sense I can remove it from this PR for now if we want to merge it and add it later